setup.py

```diff
@@ -1767,7 +1767,9 @@ def detect_expat_elementtree(self):
                 ('XML_POOR_ENTROPY', '1'),
             ]
             extra_compile_args = []
-            expat_lib = []
+            # bpo-44394: libexpact uses isnan() of math.h and needs linkage
+            # against the libm
```

Comment:
It seems like it's a recent libexpact change, in version 2.4.0: https://github.com/libexpat/libexpat/issues/511

I don't know if it's worth mentioning it.

Reply:
Nah, I think is good since is the one we use currently in all branches